vaadin-icons-flow-demo/src/main/java/com/vaadin/flow/component/icon/demo/IconView.java

```diff
@@ -92,27 +90,7 @@ private void createClickableIconsView() {
     }
 
     private void createAllIconsView() {
-        HorizontalLayout iconLayout = new HorizontalLayout();
-        iconLayout.getStyle().set("flexWrap", "wrap");
-
-        iconLayout.setDefaultVerticalComponentAlignment(Alignment.CENTER);
-
-        for (VaadinIcon icon : VaadinIcon.values()) {
-            Icon iconComponent = icon.create();
-            iconComponent.setSize("50px");
-            iconComponent.getStyle().set("color", "#00b4f0").set("marginBottom",
-                    "3px");
-            VerticalLayout iconWithName = new VerticalLayout(iconComponent,
-                    new Label(icon.name()));
-            iconWithName.setSizeUndefined();
-            iconWithName
-                    .setDefaultHorizontalComponentAlignment(Alignment.CENTER);
-            iconWithName.getStyle().set("margin", "5px").set("width", "140px")
-                    .set("fontSize", "12px");
-            iconLayout.add(iconWithName);
-        }
-
-        iconLayout.setId("all-icons");
-        addCard("All available icons", iconLayout);
+        AllIconsView allIcons = new AllIconsView();
+        add(allIcons);
```

Comment:
we should use `addCard("All available icons"` to keep the same styles as other demos
this means we should remove the header from the template

Reply:
done